controller:
    type: position_controllers/JointTrajectoryController
    joints:
       - head_joint
       - joint1
       - joint2
    constraints:
      goal_time: 0.6
      stopped_velocity_tolerance: 0.05
      head_joint: {trajectory: 0.1, goal: 0.1}
      joint1: {trajectory: 0.1, goal: 0.1}
      joint2: {trajectory: 0.1, goal: 0.1}
    stop_trajectory_duration: 0.5
    state_publish_rate:  25
    action_monitor_rate: 10
/gazebo_ros_control:
    pid_gains:
      head_joint: {p: 1000.0, i: 100.0, d: 10.0}
      joint1: {p: 1000.0, i: 100.0, d: 10.0}
      joint2: {p: 1000.0, i: 100.0, d: 10.0}
      
      
      
      